src/amo/components/HeadLinks/index.js

```diff
@@ -0,0 +1,85 @@
+/* @flow */
+import config from 'config';
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import invariant from 'invariant';
+
+import { getCanonicalURL } from 'amo/utils';
+import { hrefLangs } from 'core/languages';
+import type { AppState } from 'amo/store';
+
+type Props = {|
+  to: string,
+|};
+
+type InternalProps = {|
+  ...Props,
+  _config: typeof config,
+  _hrefLangs: typeof hrefLangs,
+  clientApp: string,
+  lang: string,
+|};
+
+export class HeadLinksBase extends React.PureComponent<InternalProps> {
+  static defaultProps = {
+    _config: config,
+    _hrefLangs: hrefLangs,
+  };
+
+  render() {
+    const { _config, _hrefLangs, clientApp, lang, to } = this.props;
+
+    if (_config.get('unsupportedHrefLangs').includes(lang)) {
+      return null;
```

Comment:
Do we still need to render the `rel="canonical"` link for this case? The old code was rendering it.

Reply:
oh 🤦‍♂️